RDF.js

```diff
@@ -1319,6 +1447,7 @@ function doImport() {
 			startImport(resolve, reject);
 		});
 	}
+	return false;
```

Comment:
Why is this needed in `doImport`?

Reply:
This is because of the new promise things use a `return`, i.e.
```
return new Promise(function (resolve, reject) ...
```
https://github.com/zotero/translators/blame/58580c4a47f7ca1b362a9d4a128da6b7a079ae02/RDF.js#L1446

@dstillman Is this the way to do that and therefore also include this `return false` here?

Reply:
That's fine — it just needs to be falsy — but it should be moved up to the first block, since it doesn't make sense after the `return new Promise`.

Reply:
Just to make sure I understand you correctly: I will move the `return false;` up as the last line inside the `if` statement. Is that right?

Reply:
Yes, that's right. (And the `else` could then be removed.)